src/operations/indexes.ts

```diff
@@ -195,8 +228,7 @@ export class CreateIndexesOperation<
 
     this.options = options ?? {};
     this.collectionName = collectionName;
-
-    this.indexes = indexes;
+    this.indexes = fillInIndexNames(indexes);
```

Comment:
Since we're actually normalizing the index key types here, should we update the `indexes` class property type to reflect that key will always be a map?

Reply:
Done!